hail/python/hail/ggplot/ggplot.py

```diff
@@ -256,10 +258,15 @@ def get_aggregation_result(selected, mapping_per_geom, precomputed):
         fig = fig.update_yaxes(title_font_size=18, ticks="outside")
         fig.update_layout(
             plot_bgcolor="white",
-            xaxis=dict(linecolor="black"),
-            yaxis=dict(linecolor="black"),
             font_family='Arial, "Open Sans", verdana, sans-serif',
-            title_font_size=26
+            title_font_size=26,
+            xaxis=dict(linecolor="black", showticklabels=True),
+            yaxis=dict(linecolor="black", showticklabels=True),
+            **{
+                f"{var}axis{idx}": {"linecolor": "black", "showticklabels": True}
+                for idx in range(2, n_facet_rows + n_facet_cols + 1)
```

Comment:
this is probably worthy of a short comment explaining the sillyness of the axis keyword argument names.

Reply:
Something like:
```
# xaxis stands in for xaxis1; xaxis2 and beyond are as expected
```

Reply:
Added a comment!